dubbo-common/src/main/java/org/apache/dubbo/common/extension/ExtensionLoader.java

```diff
@@ -624,6 +624,22 @@ private boolean isSetter(Method method) {
 
     // synchronized in getExtensionClasses
     private Map<String, Class<?>> loadExtensionClasses() {
+        cacheDefaultExtensionName();
+
+        Map<String, Class<?>> extensionClasses = new HashMap<String, Class<?>>();

```

Comment:
```suggestion
        Map<String, Class<?>> extensionClasses = new HashMap<>();
```
It looks better like this.